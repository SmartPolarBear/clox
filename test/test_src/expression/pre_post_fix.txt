R"(
var a=0;
print a++;
print ++a;
)"