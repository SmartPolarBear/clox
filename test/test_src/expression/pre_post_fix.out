R"(0.000000
2.000000)"